app/assets/stylesheets/application.css

```diff
@@ -13,3 +13,101 @@
  *= require_tree .
  *= require_self
  */
+#new_user {
```

Comment:
I'd break this out to another file. `application.css` can grow out of control and become terrible for merge conflicts if you're not careful.